src/heartbeat/heartbeat.js

```diff
@@ -8,16 +8,39 @@ let id = '';
 let serverUrl = '';
 let interval = TEN_MINUTES;
```

Comment:
interval should be shorter I think, maybe a minute is better

Reply:
Waaaaaayyyyy shorter

Reply:
Once every second I would say

Reply:
💯